packages/react-frontend/src/main.jsx

```diff
@@ -42,6 +43,10 @@ const router = createBrowserRouter([
   {
     path: "Album/:id",
     element: <AlbumInfo />
+  },
+  {
+    path: "about",
+    element: <About />
   }
   // {
   //   path: "terms-and-conditions",
```

Comment:
i commented out the terms and conditions when i was working on setting up the CI/CD because it had a lot of syntactical issues due to how long the code was haha, but i think the idea of a terms and conditions is still cool! we can still implement it but make it shorter perhaps?

Reply:
agreed, I like the idea of having a terms and conditions. It makes the website more fun it can be an easter egg kind of page

Reply:
Yes, terms and condition would be good! The website I used to make the T&C just caused a lot of syntax errors as you said, so i might have to find a work around!

Reply:
Update: Terms and Conditions Page is up!